.github/actions/setup-miniconda/action.yml

```diff
@@ -116,7 +116,6 @@ runs:
             ${ENV_FILE_FLAG} \
             python="${PYTHON_VERSION}" \
             cmake=3.22 \
-            conda-build=3.21 \
```

Comment:
Nice, could we also remove `conda install --yes -p ${CONDA_ENV} conda-build=3.21` on line 144.  The package `conda-build` hardcodes the path and couldn't be cached